content/release_notes_chefdk.md

```diff
@@ -2580,7 +2580,7 @@ cookbook(s) given on the command line.
 - **Added include_policy directive**
 
     Policyfile can use the `include_policy` directive as described in
-    [RFC097](https://github.com/chef/chef-rfc/blob/master/rfc097-policyfile-includes.md).
+    [RFC097](https://github.com/chef/chef-rfc/blob/main/rfc097-policyfile-includes.md).
```

Comment:
chef-rfc is in the boneyard, so it is still `master`

see: https://github.com/chef/chef-rfc